exchange/non_bid_reason.go

```diff
@@ -6,7 +6,8 @@ package exchange
 type NonBidReason int
 
 const (
-	NoBidUnknownError                      NonBidReason = 0   // No Bid - General
+	NoBidUnknownError                      NonBidReason = 0 // No Bid - General
+	ResponseRejectedGeneral                NonBidReason = 300
```

Comment:
For my own understanding: Why is the non bid reason not specific to referencing DSA?

Reply:
The non bid reason codes are defined in the seat nonbid community extension documentation, section [List: Non Bid Status Codes](https://github.com/InteractiveAdvertisingBureau/openrtb/blob/main/extensions/community_extensions/seat-non-bid.md#list-non-bid-status-codes). There isn't a DSA specific code at this time so we agreed to use the generic bid response rejected 300 code for now.

Reply:
Actually we agreed that the status code should now be 305, Response Rejected - Privacy, so I pushed an update.